### PR TITLE
migrations integration on front end

### PR DESCRIPTION
**Pull request by piotrwitek**
Added migrations integration on portfolio page
Added migrations specific feature flag
Screenshot 2024-01-31 at 13 52 53

### CODE REVIEW

File: components/portfolio/PortfolioOverview.tsx [hunk 1]
```diff
@@ -74,9 +84,22 @@ export const PortfolioOverview = ({
...
+              </Heading>
+            ) : (
+              <Tag sx={{ mt: 2 }} variant="tagInteractive">
+                {tPortfolio('coming-soon')}
+              </Tag>
+            )
+          }
+          subValue={
+            getLocalAppConfig('features').EnableMigrations && (
+              <AppLink href={'TODO migration link'} sx={{ mr: 3 }}>
```
Comment (jakubswierczek) [inline components/portfolio/PortfolioOverview.tsx:99]: Could you add a story for link handling?

File: components/portfolio/PortfolioOverview.tsx [hunk 2]
```diff
@@ -14,14 +18,20 @@ import type { PortfolioAssetsResponse, PortfolioOverviewResponse } from './types
 export const PortfolioOverview = ({
   overviewData,
   portfolioWalletData,
+  migrationPositions,
 }: {
   overviewData: PortfolioOverviewResponse
   portfolioWalletData: PortfolioAssetsResponse
+  migrationPositions?: PortfolioPosition[] | void
```
Comment (jakubswierczek) [inline components/portfolio/PortfolioOverview.tsx:25]: `Void` is not necessary. The property is optional, so it's undefined when not provided.

File: pages/portfolio/[address].tsx [hunk 3]
```diff
@@ -67,6 +69,15 @@ export default function PortfolioView(props: PortfolioViewProps) {
 
   const { address, awsInfraUrl, awsInfraHeader } = props
 
+  // loading migrations on load
+  const [migrationPositions, setMigrationPositions] = React.useState<PortfolioPosition[] | void>(
```
Comment (jakubswierczek) [inline pages/portfolio/[address].tsx:73]: `PortfolioPosition[] | void` -> `PortfolioPosition[] | undefined`

**Review by jakubswierczek — CHANGES_REQUESTED** [review]

File: components/portfolio/PortfolioOverview.tsx (hunk 2 above)
Comment (piotrwitek) [reply components/portfolio/PortfolioOverview.tsx:25]: done

File: pages/portfolio/[address].tsx (hunk 3 above)
Comment (piotrwitek) [reply pages/portfolio/[address].tsx:73]: done

File: components/portfolio/PortfolioOverview.tsx (hunk 1 above)
Comment (piotrwitek) [reply components/portfolio/PortfolioOverview.tsx:99]: https://app.shortcut.com/oazo-apps/story/14173

**Review by jakubswierczek — APPROVED** [review]

**Pull request merged**